src/api/actions/initialize_gsuite.ts

```diff
@@ -0,0 +1,59 @@
+import * as jwt from "jsonwebtoken";
+import axios from "axios";
+import * as Sentry from "@sentry/react";
+import { decoded_jwt, auth_status_gsuite } from "../../config/interface";
+
+const verify_gsuite = async (authToken: string): Promise<auth_status_gsuite> => {
+
+  if (authToken === undefined || authToken === "") {
+    return {
+      jwt: authToken,
+      is_verified: false,
+    };
+  }
+
+  // await new Promise(resolve => setTimeout(resolve, 5000));
+  const decodedToken: decoded_jwt = (jwt.decode(authToken, {
+    complete: true,
+  }) as any).payload;
+  const user_email = decodedToken["https://acmutd.co/email"];
```

Comment:
The jwt from cloudflare access doesn't have this field. One of the custom scripts around authentication from Auth0 handles injecting this field in. If you want to see what fields a jwt contains you can copy and paste it into [jwt.io](https://jwt.io). 

You'll need to go to the site about to find out what the field containing the email is called and update the `decodedToken["stuff-here"]` to have the appropriate field name.

Reply:
Looks like the field is just called email.